myapp/lib/logic.dart

```diff
@@ -1,6 +1,29 @@
+// import 'dart:core';
+
+class Data{
+  double value;
+  String msg;
+
+  Data({this.value = 0.0, this.msg = 'UnderWeight'});
+}
+
 class Logic{
-  static double compute({required double height, required double weight}){
+  static Data compute({required double height, required double weight}){
```

Comment:
Have a request model and pass the object here, instead of directly sharing the height and width

Reply:
Created two classes (Request and Response) for handling requests and responses between UI and logic